core/src/test/java/org/apache/calcite/test/SqlValidatorTest.java

```diff
@@ -7033,7 +7036,7 @@ void testGroupExpressionEquivalenceParams() {
     sql("select cast(a as row(f0 int, f1 varchar)) from COMPLEXTYPES.CTC_T1")
         .withExtendedCatalog()
         .columnType("RecordType(INTEGER NOT NULL F0, VARCHAR NOT NULL F1) NOT NULL");
-    sql("select cast(b as row(f0 int not null, f1 varchar null))\n"
+    sql("select cast(b as row(f0 int, f1 varchar))\n"
```

Comment:
why are you changing this test and the below one?

Reply:
just for test row and i fallback it.
then i will submit new pr to fix cast( type not null/ null) latter according sql standard, what do you think?

Reply:
sounds good to me, one commit should focus on one thing